external-crates/move/crates/move-package/src/lib.rs

```diff
@@ -96,6 +94,10 @@ pub struct BuildConfig {
     /// If set, warnings become errors
     #[clap(long = move_compiler::command_line::WARNINGS_ARE_ERRORS, global = true)]
     pub warnings_are_errors: bool,
+
+    /// Additional named address mapping. Useful for tools in rust
+    #[clap(skip)]
+    pub additional_named_addresses: BTreeMap<String, AccountAddress>,
```

Comment:
TOML serialization fails if serializable tables are not at the end of the data structure, so I moved `additional_named_addresses` to the end. This means a bunch of test output had to be updated (`Move.compiled` and `Move.resolved` files).

There exists a macro [toml::ser::tables_last](https://teaclave.apache.org/api-docs/crates-enclave/toml/ser/fn.tables_last.html#) but it didn't work for me no matter what I tried, so I ended up moving this value in the struct.

Note: I could have avoided this by manually serializing the `BuildConfig` data structure, but I think that's a bad idea because it's going to be easy for anyone who changes this data structure to forget to update the manual serial business. I really think capturing all of `BuildConfig` is pretty necessary to stop footguns.